<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SETDIMDIALOG</class>
 <widget class="QDialog" name="SETDIMDIALOG">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>273</width>
    <height>79</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>273</width>
    <height>79</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>273</width>
    <height>79</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="increase_radio_button">
         <property name="text">
          <string>Increase</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="reduce_radio_button">
         <property name="text">
          <string>Reduce</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="set_radio_button">
         <property name="text">
          <string>Set</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QSpinBox" name="spinBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>31</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="change_button">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>31</height>
          </size>
         </property>
         <property name="text">
          <string>Change</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exit_button">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>31</height>
          </size>
         </property>
         <property name="text">
          <string>Exit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
